pkg/middleware/ratelimiter/examples/http_middleware.go/http_middleware.go

```diff
@@ -0,0 +1,90 @@
+package main
```

Comment:
Can you added `pkg/middleware/ratelimiter/examples/http_middleware.go/` to the sonar exclude list so it's not included in the checks?